rdr_service/dao/workbench_dao.py

```diff
@@ -1169,6 +1172,30 @@ def _insert_history(session, researchers):
                 affiliations_history.append(affiliation_obj)
             history.workbenchInstitutionalAffiliations = affiliations_history
             session.add(history)
+            hist_researchers.append(history)
+        session.flush()
+        session.commit()
+
+        # Generate tasks to build PDR records.
+        if GAE_PROJECT == 'localhost':
+            rebuild_bq_wb_researchers(hist_researchers)
+        else:
```

Comment:
Do we need to limit how many records are rebuilt per task?

Reply:
I have added a 'list_hhunks()' for loop to limit batch size.